dao/src/main/java/greencity/entity/Position.java

```diff
@@ -0,0 +1,26 @@
+package greencity.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@Entity
+@EqualsAndHashCode
+@Table(name = "positions")
+public class Position {
```

Comment:
can we use enum for `Position`?

Reply:
I think it would be better not to move it to an enum.

Reply:
1. In order not to violate the logic of the existing functionality.
2. Because we can create new positions and tie them to rights.
3. Rewriting this piece of code will take a very long time.